shared/javascripts/local_storage.js

```diff
@@ -87,14 +87,20 @@ var ls = {
   }
 };
 
-// Determine whether localstorage can be used directly
-try { 
-  localStorage;
+try {
+  // check for jetpack runtime environment
+  exports.ls = ls;
```

Comment:
exports is not defined.
[exports](https://developer.mozilla.org/en-US/Add-ons/SDK/Guides/Module_structure_of_the_SDK) is a special variable available that allows me to reuse this module in Jetpack.
If this script is executed in a normal/privileged js runtime(i.e, from privly applications), then this would be undefined, though if someone defines exports, it could cause problems.